helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -27,19 +27,20 @@ data:
         loadbalance {{ .Values.loadbalancePolicy }}
         prometheus :{{ .Values.ports.prometheus }}
 
-        {{- if .Values.configmap.proxy }}
-        {{- range (.Values.configmap.proxy | trimAll "\n " |  split "\n") }}
-        proxy {{ . }}
-        {{- end }}
-        {{- end }}
-
-        {{- if .Values.configmap.custom }}
-        {{- range (.Values.configmap.custom | trimAll "\n " |  split "\n") }}
-        {{ . }}
-        {{- end }}
+        {{- if .Values.configmap.forward }}
+        forward . {{ range (.Values.configmap.forward | trimAll "\n " |  split "\n") }}{{ . }}{{ end }} /etc/resolv.conf {
+          policy sequential
+        }
```

Comment:
I ensure order when use define upstreams

Reply:
There is a limit of 15 upstream for the `forward` directive. It would be good to have this documented.